Unified/checkor.py

```diff
@@ -158,7 +158,7 @@ def time_point(label="",sub_lap=False, percent=None):
                     keyword = c.body[(c.body.find(force_complete_jira_string)+len(force_complete_jira_string)):].split()[0]
                     if keyword and user in actors:
                         print user,"is force-completing", keyword,"from JIRA"
-                        bypasses.update( keyword )
+                        bypasses.extend( keyword )
```

Comment:
because 
>>> l=range(10)
>>> l.extend('keyword')
>>> l
[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 'k', 'e', 'y', 'w', 'o', 'r', 'd']
>>>

it should be append, and not extend

>>> l=range(10)
>>> l.append('keyword')
>>> l
[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 'keyword']

you are lucky we have been through https://github.com/CMSCompOps/WmAgentScripts/commit/f94c63ce4ade9c505386e8d1ca622654ec14bcc6 in the past, otherwise all current workflows in assistance would have been bypassed

Reply:
thank you for checking this. would have been really bad.